<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>HostDialogUI</class>
 <widget class="QDialog" name="HostDialogUI" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>285</width>
    <height>295</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Modify Host</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>4</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QLabel" name="textLabel1" >
       <property name="text" >
        <string>Name:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="FieldLineEdit" name="mNameEdit" >
       <property name="sizePolicy" >
        <sizepolicy>
         <hsizetype>3</hsizetype>
         <vsizetype>4</vsizetype>
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="field" >
        <string>name</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="mOnlineCheck" >
     <property name="text" >
      <string>Online</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="mAllowMappingCheck" >
     <property name="text" >
      <string>Allow Automatic Mapping(Assburner)</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="mAllowSleepCheck" >
     <property name="text" >
      <string>Allow Automatic Sleeping(Assburner)</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>2</number>
     </property>
     <item row="5" column="0" >
      <widget class="QLabel" name="label" >
       <property name="text" >
        <string>Serial #:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" >
      <widget class="QLabel" name="textLabel8" >
       <property name="text" >
        <string>OS:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" >
      <widget class="QLabel" name="textLabel6" >
       <property name="text" >
        <string>Mhz:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0" >
      <widget class="QLabel" name="textLabel2" >
       <property name="text" >
        <string>CPUs:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0" >
      <widget class="QLabel" name="textLabel5" >
       <property name="text" >
        <string>Memory(MB):</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" >
      <widget class="QLabel" name="textLabel3" >
       <property name="text" >
        <string>Description:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" >
      <widget class="FieldLineEdit" name="mOSEdit" >
       <property name="sizePolicy" >
        <sizepolicy>
         <hsizetype>7</hsizetype>
         <vsizetype>4</vsizetype>
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="field" >
        <string>os</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" >
      <widget class="FieldLineEdit" name="mDescriptionText" >
       <property name="field" >
        <string>description</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1" >
      <widget class="QSpinBox" name="mMemorySpin" >
       <property name="maximum" >
        <number>99999999</number>
       </property>
      </widget>
     </item>
     <item row="2" column="1" >
      <widget class="QSpinBox" name="mMhzSpin" >
       <property name="maximum" >
        <number>999999999</number>
       </property>
      </widget>
     </item>
     <item row="5" column="1" >
      <widget class="FieldLineEdit" name="mSerialEdit" >
       <property name="field" >
        <string>sn</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1" >
      <widget class="QSpinBox" name="mCpusSpin" >
       <property name="maximum" >
        <number>999999</number>
       </property>
       <property name="minimum" >
        <number>1</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QPushButton" name="mServiceButton" >
       <property name="text" >
        <string>Edit Services</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mInterfacesButton" >
       <property name="text" >
        <string>Edit Interfaces</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="mOKButton" >
       <property name="text" >
        <string>&amp;OK</string>
       </property>
       <property name="shortcut" >
        <string>Alt+O</string>
       </property>
       <property name="default" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mCancelButton" >
       <property name="text" >
        <string>&amp;Cancel</string>
       </property>
       <property name="shortcut" >
        <string>Alt+C</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <customwidgets>
  <customwidget>
   <class>FieldLineEdit</class>
   <extends>QLineEdit</extends>
   <header>fieldlineedit.h</header>
   <container>0</container>
   <pixmap></pixmap>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>mNameEdit</tabstop>
  <tabstop>mOnlineCheck</tabstop>
  <tabstop>mDescriptionText</tabstop>
  <tabstop>mOSEdit</tabstop>
  <tabstop>mSerialEdit</tabstop>
  <tabstop>mServiceButton</tabstop>
  <tabstop>mInterfacesButton</tabstop>
  <tabstop>mOKButton</tabstop>
  <tabstop>mCancelButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>mOKButton</sender>
   <signal>clicked()</signal>
   <receiver>HostDialogUI</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>349</x>
     <y>595</y>
    </hint>
    <hint type="destinationlabel" >
     <x>284</x>
     <y>595</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mCancelButton</sender>
   <signal>clicked()</signal>
   <receiver>HostDialogUI</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>460</x>
     <y>591</y>
    </hint>
    <hint type="destinationlabel" >
     <x>459</x>
     <y>609</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
